app/gui/view/examples/execution-environment-dropdown/src/lib.rs

```diff
@@ -19,32 +19,33 @@ use ensogl::prelude::*;
 use ensogl::animation;
 use ensogl::application::Application;
 use ensogl_text_msdf::run_once_initialized;
-use ide_view_execution_mode_selector as execution_mode_selector;
-
+use ide_view_execution_environment_selector as execution_environment_selector;
+use ide_view_execution_environment_selector::ExecutionEnvironments;
 
 
 // ======================
 // === Initialisation ===
 // ======================
 
-fn make_entries() -> execution_mode_selector::ExecutionModes {
-    Rc::new(vec!["development".to_string(), "production".to_string()])
+fn make_entries() -> ExecutionEnvironments {
+    Rc::new(vec!["Design".to_string().into(), "Live".to_string().into()])
```

Comment:
another hardcoded place with the modes. I would like to ask that all mode names will be defined ONLY in one place in the code.

Reply:
We don't want all of them in one place only because of crate dependencies: we have a simple demo scene for a chooser that can take arbitrary labels. This element and its demo scene do not need to know about the language server protocol that contains our current  `ExecutionEnvironment` and its valid values. So, it makes sense to just use fixed dummy values there. We can unify those dummy values between the two demo scenes, but I would argue against introducing a dependency on the LS protocol.